### PR TITLE
Add total row to mobile UCR

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?188969

@orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/fixtures/mobile_ucr.py [hunk 1]
```diff
@@ -84,14 +88,41 @@ def _report_config_to_fixture(self, report_config, user):
...
+            rows_elem.append(_row_to_row_elem(row, i))
+
+        if data_source.has_total_row:
+            total_row = get_total_row(
+                data_source.get_data(),
+                data_source.aggregation_columns,
+                data_source.column_configs,
+                get_expanded_columns(data_source.column_configs, data_source.config)
+            )
+            rows_elem.append(_row_to_row_elem(
```
Comment (nickpell) [inline corehq/apps/app_manager/fixtures/mobile_ucr.py]: going to revisit the implementation of ```get_total_row soon```, and likely will refactor this code at that point

File: corehq/apps/app_manager/tests/test_report_config.py [hunk 2]
```diff
@@ -209,7 +209,7 @@ def test_graph(self):
         <partial>
           <template form="graph">
             <graph type="bar">
-              <series nodeset="instance('reports')/reports/report[@id='a98c812873986df34fd1b4ceb45e6164ae9cc664']/rows/row[column[@id='computed_owner_name_40cc88a0']=instance('commcaresession')/session/data/report_filter_a98c812873986df34fd1b4ceb45e6164ae9cc664_computed_owner_name_40cc88a0_1]">
+              <series nodeset="instance('reports')/reports/report[@id='a98c812873986df34fd1b4ceb45e6164ae9cc664']/rows/row[@is_total_row='False'][column[@id='computed_owner_name_40cc88a0']=instance('commcaresession')/session/data/report_filter_a98c812873986df34fd1b4ceb45e6164ae9cc664_computed_owner_name_40cc88a0_1]">
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_report_config.py]: E501 line too long (318 > 115 characters)

**Comment by orangejenny** [comment]
Looks reasonable. Merging, but do you think it's worthwhile to add a test that has a total row?

File: corehq/apps/app_manager/fixtures/mobile_ucr.py (hunk 1 above)
Comment (czue) [inline corehq/apps/app_manager/fixtures/mobile_ucr.py]: :+1: this function seems to be growing outside of what it should ideally be responsible for

**Pull request merged**